crates/cxx-qt-lib/build.rs

```diff
@@ -187,6 +188,10 @@ fn main() {
         cxx_build::CFG
             .exported_header_dirs
             .push(include_path.as_path());
+        println!(
```

Comment:
this looks like a trace line ?

Reply:
I used that to debug includes, of course. But isn't useful information ? Maybe in some other form ?

Reply:
fixed in the next comment